### DISCUSSION TITLE
Shortcut to turn the current drawing tool in and out

### DISCUSSION THREAD

**Discussion by kacago22** [xournalpp/xournalpp#6146] [Ideas]
The drawing tool can be switched on and off with Ctrl+n - where n is corresponding to the current drawing tool. It would be nice if we could use Ctrl+0 to turn the current drawing tool in and out - that way I could even assign a button from my drawing tablet to it.

I do teach math in school with xournal++ for two years now and (not) turning on and off the drawing is the most annoying mistake that keeps repeating over and over!

**Comment by MiltonBalaOfficial** [comment]
A plugin may solve your issue, (I have developed a plugin for similar kind of issue) If you share your problems in detail, I may share my plugin with some modification if needed.
